### PR TITLE
fix(model): align fields of ohshown event type with frontend

### PR DESCRIPTION
**Pull request by tai271828**
As-is Issue: #15
To-be:

Selection_625

## Test This Pull Request
Go to the admin console --> `Ohshown Events` --> choose any reported entry --> check the draw-down list of ohshown event type

## Additional Information
The fix is way more straightforward than I expected. We can improve this feature via issue #22

### CODE REVIEW

**Comment by tai271828** [comment]
Oops CI failed. Fixing.

**Comment by tai271828** [comment]
CI is fixed. This pull request is ready to review. If you have a moment, I will appreciate your review.

File: backend/api/models/ohshown_event.py [hunk 1]
```diff
@@ -13,16 +13,9 @@ class OhshownEvent(SoftDeleteMixin):
...
-        ("2-2", "焊接、鑄造、熱處理"),
-        ("2-3", "金屬表面處理、噴漆"),
-        ("3", "塑膠加工、射出"),
-        ("4", "橡膠加工"),
-        ("5", "非金屬礦物（石材）"),
-        ("6", "食品"),
-        ("7", "皮革"),
-        ("8", "紡織"),
-        ("9", "其他"),
+        ("2-1", "目擊黑熊"),
```
Comment (EagleC0318) [inline backend/api/models/ohshown_event.py:16]: Looks like it's different (a simplified version) with the UI dropped-down list (according to the screenshot in https://github.com/tai271828/disfactory-backend/issues/15).
Is it good enough for now?

Comment (EagleC0318) [reply backend/api/models/ohshown_event.py:16]: And FYI that I have made the dropped-down list in the PR https://github.com/tai271828/disfactory-backend/pull/18.
It was for the get event types API endpoint to synchronize the dropped-down list between the BE and FE.
Maybe we should reuse/integrate this list within this PR.

Comment (tai271828) [reply backend/api/models/ohshown_event.py:16]: The elements are aligned with the latest frontend drop-down list. See the development site (which is deployed with the latest codebase)

> Looks like it's different (a simplified version) with the UI dropped-down list (according to the screenshot in #15). Is it good enough for now?

**Pull request merged**